### ISSUE TITLE
aggregate-subtractor-fn not in willa.experiment

### ISSUE THREAD

**Issue by jeremyplichtafc** [DaveWM/willa#6]
When running an experiment in willa where I have a ktable that has a key added to it then removed to it the `aggregate-subtractor-fn` I supply does not seem to get called. When looking in the willa.experiment namespace I see references to `aggregate-adder-fn` though not to `aggregate-subtractor-fun`. Does the experiment module currently not support where you add a key to a ktable then remove it it?

**Comment by DaveWM** [comment]
You're right, I haven't added support for `aggregate-subtractor-fn` to experiments. I should have some time this weekend, I'll try to get it done then
